src/lib.rs

```diff
@@ -13,6 +13,12 @@
 use ansi_width::ansi_width;
 use std::fmt;
 
+/// Number of spaces in one \t.
```

Comment:
Minor improvement is to put the character between backticks to make it nicer in rustdoc: `` `\t` ``

This only goes for the doc comments of course.